UIKitDZ/ModelForSong/ExtensionToBookViewController.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ExtensionToBookViewController.swift
+//  UIKitDZ
+//
+//  Created by coder on 27.09.2022.
+//
+
+import Foundation
+import UIKit
+/// ExtensionToBookViewController - расширяем наш класс для использование пикеров
```

Comment:
```suggestion
/// UIPickerViewDelegate, UIPickerViewDataSource
```
этого достаточно, чтобы понять что внутри